source/index.html.md

```diff
@@ -18,7 +18,7 @@ It's useful to have some understanding of how RESTful APIs use HTTP verbs such a
 
 ## Quick start guide and examples
 
-The GOV.UK Content API can be interrogated using the command line utilities `curl` or `wget`. These provide a quick way of seeing the JSON API responses prior to developing more complex integrations.
+The GOV.UK Content API can be interrogated using the command line utilities `curl` or `wget`. These provide a quick way of viewing JSON API responses prior to developing more complex integrations.
```

Comment:
interrogated? sounds quite draconian hehe

Reply:
Agree it's a bit _big brother_ for government documentation, will use a less invasive term.

Reply:
Sure - I'm having a bit of a ponder about to rework this section a bit to be a sequence of quick steps (just as a warning)